backports/ceres-solver/ENABLE_ON

```diff
@@ -1 +1,2 @@
 v3.14
+v3.17
```

Comment:
We can remove ENABLE_ON file

Reply:
Done in 7c55b74c370fb7c9d93e40a05a8d8bffe5bf96da